frontend/src/tests/components/eMIB/ActionViewEmail.test.js

```diff
@@ -148,3 +148,15 @@ function createWrapper(responseType, cc, deleteEmail) {
     />
   );
 }
+
+it("it renders when the email is undefined", () => {
+  shallow(
+    <UnconnectedActionViewEmail
+      actionId={0}
+      action={{ actionType: ACTION_TYPE.email }}
+      email={emailStub}
```

Comment:
but email is defined here?

Reply:
as mentioned offline, "email" here is the original email that is being replied to; "action" is the response, which always has a type assigned.